reijnaers2021re/openreview/packages.tex

```diff
@@ -0,0 +1,30 @@
+%\usepackage[utf8]{inputenc}
+
+\usepackage{biblatex}
+\addbibresource{bibliography.bib}
+
+\usepackage[T1]{fontenc} 
+\usepackage{hyperref}
+\usepackage{url}
+\usepackage{booktabs}
+\usepackage{amsfonts}
+\usepackage{nicefrac}
+\usepackage{microtype}
+\usepackage[acronym]{glossaries}
+\usepackage{graphicx}
+\usepackage{float}
+\usepackage{wrapfig}
+\usepackage{pgf}
+\usepackage{pgfplots}
+\usepackage{acronym}
+\usepackage{caption}
+\usepackage{subcaption}
+\usepackage{bbold}
+\usepackage{changepage}
+\usepackage{adjustbox}
+\usepackage{appendix}
+\usepackage{eurosym}
+\usepackage{standalone}
+
```

Comment:
You use `subfloat` in the content, which needs package `subfig` to be imported here:

```
\usepackage{subfig}
```

Reply:
Also remove the subcaption package while you are at it (will not work with subfig)

Reply:
Thank you for spotting this. Fixed!